guides/v2.0/test/unit/unit_test_execution_phpstorm.md

```diff
@@ -11,7 +11,7 @@ functional_areas:
   - test
 ---
 
-Running tests in PHPStorm requires a couple steps to setup the system.
+Running tests in PHPStorm requires following steps to setup the system.
```

Comment:
This sentence needs an article between "requires" and "following". I also think it can be simplified. For example,
>Running tests in PHPStorm requires **the** following steps: